onnx/defs/reduction/utils.cc

```diff
@@ -9,24 +9,26 @@
 #include <vector>
 
 namespace ONNX_NAMESPACE {
-std::vector<std::string> GetSupportedDataTypesForReductionOps(bool supports8bit) {
+std::vector<std::string> GetSupportedDataTypesForReductionOps(bool supports8bit, bool supports_bool) {
+  auto data_types = OpSchema::numeric_types_for_math_reduction_ir4();
   if (supports8bit) {
-    auto data_types = OpSchema::numeric_types_for_math_reduction_ir4();
     data_types.push_back("tensor(uint8)");
     data_types.push_back("tensor(int8)");
-
-    return data_types;
+  }
+  if (supports_bool) {
+    data_types.push_back("tensor(bool)");
   }
 
-  return OpSchema::numeric_types_for_math_reduction_ir4();
+  return data_types;
 }
 
-std::function<void(OpSchema&)> ReduceDocGenerator_opset13_18(
+std::function<void(OpSchema&)> ReduceDocGenerator_opset13_20(
```

Comment:
It may be useful if the documentation mentions that "false < true" in computing min/max. It's probably a bit involved given that the documentation is shared by many ops in different opsets.

Reply:
Updated to the following:


Computes the min of the input tensor's elements along the provided axes. The resulting
tensor has the same rank as the input if `keepdims` equals 1. If `keepdims` equals 0, then
the resulting tensor has the reduced dimension pruned. Input tensors of rank zero are
valid.

If the input data type is Boolean, the comparison should consider `False < True`.

The above behavior is similar to numpy, with the exception that numpy defaults keepdims to False
instead of True.